src/AutoRest.CSharp/MgmtTest/Generation/MgmtBaseTestWriter.cs

```diff
@@ -579,7 +579,7 @@ protected override void WriteResourceCollectionEntry(ResourceCollection resource
             var parameterValues = new List<KeyValuePair<string, FormattableString>>();
             foreach (var passThruParameter in methodParameters)
             {
-                if (passThruParameter.Name == MgmtClientOperation.WaitForCompletionParameter.Name ||
+                if (passThruParameter.Name == KnownParameters.WaitForCompletion.Name ||
                     passThruParameter.Name == MgmtClientBaseWriter.CancellationTokenParameter.Name)
```

Comment:
This might also need to change if we change the cancellationTokenParameter to use the common one as well